README.md

```diff
@@ -12,6 +12,12 @@ A component of the [platform](https://corchestra.ru/en/) for rapid development o
 
 * [JavaDoc](https://courseorchestra.github.io/celesta/apidocs)
 
+## See Also
+
+* [DBSchemaSync](https://github.com/CourseOrchestra/dbschemasync): Celesta -- DBSchema sync utility.
```

Comment:
Please also add a reference to [CelestaDoc](https://github.com/CourseOrchestra/celestadoc) -- CelestaSQL database documentation generator

Reply:
Done.